calyx/src/ir/printer.rs

```diff
@@ -357,6 +358,9 @@ impl Printer {
                         Self::port_to_str(&port.borrow())
                     )?;
                 }
+                if !external_cells.is_empty() {
```

Comment:
Let’s not change the printer in this PR. You can add an underscore or use .. to ignore unused fields in the match above.